src/Microsoft.DotNet.Wpf/src/PresentationBuildTasks/Microsoft/Build/Tasks/Windows/FileClassifier.cs

```diff
@@ -82,7 +82,7 @@ public override bool Execute()
 
             try
             {
-                TaskHelper.DisplayLogo(Log, SR.Get(SRID.FileClassifierTask));
+                TaskHelper.DisplayLogo(Log, FileClassifierTaskString);
```

Comment:
Based on our conversation, you are going to change these to `nameof(FileClassifier)` etc., right?